src/lib.rs

```diff
@@ -126,16 +119,12 @@ impl From<GlobError> for std::io::Error {
 }
 
 impl std::fmt::Display for GlobError {
-    fn fmt(&self, f: &mut std::fmt::Formatter) -> Result<(), std::fmt::Error> {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> Result<(), std::fmt::Error> {
```

Comment:
in modern Rust, it's deprecated to leave out hidden lifetimes and it's preferred to add an anonymous one